scania-digital-design-system/sdds
# AB#873 Bug - slot button react

**Describe pull-request**  
Added modal styling for CSS only'

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue_
Fixes: #158  [AB#873](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/873)

**How to test**  
_Add description how to test if possible_
1. Go to storybook
2. Check in Component > Modal > Modal CSS
